content/en/docs/concepts/scheduling-eviction/api-eviction.md

```diff
@@ -6,14 +6,117 @@ weight: 70
 
 {{< glossary_definition term_id="api-eviction" length="short" >}} </br>
 
-You can request eviction by directly calling the Eviction API 
-using a client of the kube-apiserver, like the `kubectl drain` command. 
-This creates an `Eviction` object, which causes the API server to terminate the Pod. 
+You can request eviction by calling the Eviction API directly, or programmatically
+using a client of the {{<glossary_tooltip term_id="kube-apiserver" text="API server">}}, like the `kubectl drain` command. This
+creates an `Eviction` object, which causes the API server to terminate the Pod.
 
 API-initiated evictions respect your configured [`PodDisruptionBudgets`](/docs/tasks/run-application/configure-pdb/)
 and [`terminationGracePeriodSeconds`](/docs/concepts/workloads/pods/pod-lifecycle#pod-termination). 
 
+Using the API to create an Eviction object for a Pod is like performing a
+policy-controlled [`DELETE` operation](/docs/reference/kubernetes-api/workload-resources/pod-v1/#delete-delete-a-pod)
+on the Pod. 
+
+## Calling the Eviction API
+
+You can use a [Kubernetes language client](/docs/tasks/administer-cluster/access-cluster-api/#programmatic-access-to-the-api)
+to access the Kubernetes API and create an `Eviction` object. To do this, you
+POST the attempted operation, similar to the following example:
+
+{{< tabs name="Eviction_example" >}}
+{{% tab name="policy/v1" %}}
+{{< note >}}
+`policy/v1` Eviction is available in v1.22+. Use `policy/v1beta1` with prior releases.
```

Comment:
nit: specify Kubernetes v1.22
`policy/v1` Eviction is available since Kubernetes version 1.22.